src/ontology/imports/dental_material_import.owl

```diff
@@ -386,7 +388,7 @@
     <!-- http://purl.obolibrary.org/obo/OHD_0000034 -->
 
     <Class rdf:about="http://purl.obolibrary.org/obo/OHD_0000034">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OHD_0000048"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/imports/dental_material_import.owl#direct"/>
```

Comment:
You need to revert this and keep the OHD IRI: `http://purl.obolibrary.org/obo/OHD_0000048`.  
Your classes will be added under the top level OHD classes.